src/main/java/org/example/cli/DeliveryEmployee.java

```diff
@@ -0,0 +1,20 @@
+package org.example.cli;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+// DeliveryEmployee will extend/inherit from Employee class
+public class DeliveryEmployee extends Employee {
+
+    // Constructor for delivery employee, inherits all attributes from parent Employee class
+    @JsonCreator
+    public DeliveryEmployee( @JsonProperty("id") int id,
+                             @JsonProperty("first_name") String first_name,
+                             @JsonProperty("last_name") String last_name,
+                             @JsonProperty("salary") double salary,
+                             @JsonProperty("bank_account_number") String bank_account_number,
+                             @JsonProperty("ni_number") String ni_number) {
```

Comment:
The variables aren't the normal naming convention for Java variables

Reply:
Variable names have now been updated here too :)